### PR TITLE
For #4457: Scale icon for action button in custom tabs.

### PR DESCRIPTION
**Pull request by mcarare**
---

### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### CODE REVIEW

File: components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt [hunk 1]
```diff
@@ -252,4 +258,9 @@ class CustomTabsToolbarFeature(
             }
         }
     }
+
+    companion object {
+        private const val ACTION_BUTTON_WIDTH_DP = 24
```
Comment (Amejia481) [inline components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt:263]: @mcarare should the minimum [touch target size be 48dp](https://support.google.com/accessibility/android/answer/7101858?hl=en)?

**Review by jonalmeida — COMMENTED** [review]
LGTM! Leaving it for Arturo to approve since there is an open question there.

File: components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt (hunk 1 above)
Comment (mcarare) [reply components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt:263]: This is just the size of the drawable. The view still has the same dimensions. I will change the naming to be more clear.

**Comment by jonalmeida** [comment]
@mcarare FYI the 'For' keyword doesn't automatically close the linked issue, you can use 'close' or any of the [other keywords](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) to do that. 🙂 

I've manually linked this PR so it should auto-close when it's merged.

**Comment by mcarare** [comment]
@jonalmeida I know it does not. I use it because almost all issues in Fenix have to be verified by QA before they are closed. And I manually close all AC issues linked with a Fenix issue once the Fenix issue is verified as fixed by QA.

File: components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt (hunk 1 above)
Comment (Amejia481) [reply components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt]: Thanks for the clarification 👍

**Review by Amejia481 — APPROVED** [review]

File: components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt (hunk 1 above)
Comment (mcarare) [reply components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt]: Thanks for the review!

**Comment by mcarare** [comment]
> LGTM! Leaving it for Arturo to approve since there is an open question there.
> 
> This is worthy of a changelog entry though.

I added the changelog entry.

**Pull request merged**